contribs/gmf/src/directives/search.js

```diff
@@ -517,6 +533,42 @@ gmf.SearchController.prototype.createSearchCoordinates_ = function(view) {
 
 
 /**
+ * Style for search results.
+ * @param {null|ol.Feature|ol.render.Feature} feature The searched feature.
+ * @param {number} resolution The current resolution of the map.
+ * @return {ol.style.Style} A style for this kind of features.
+ * @private
+ */
+gmf.SearchController.prototype.getSearchStyle_ = function(feature, resolution) {
+  var styles = {};
+  styles[gmf.COORDINATES_LAYER_NAME] = new ol.style.Style({
+    image: new ol.style.RegularShape({
+      stroke: new ol.style.Stroke({color: [0, 0, 0, 0.7], width: 2}),
+      points: 4,
+      radius: 8,
+      radius2: 0,
+      angle: 0
+    })
+  });
+  var fill = new ol.style.Fill({color: [255, 255, 255, 0.6]});
+  var stroke = new ol.style.Stroke({color: [60, 150, 200, 1], width: 2});
+  styles['default'] = new ol.style.Style({
+    fill: fill,
+    stroke: stroke,
+    image: new ol.style.Circle({radius: 5, fill: fill, stroke: stroke})
+  });
+  var customStyles = this.scope_['featuresStyles'] || {};
+  goog.object.extend(styles, customStyles);
+  var defaultStyle = styles['default'];
+  if (feature === null) {
+    return defaultStyle;
+  }
+  var kind = feature.get('layer_name');
+  return styles[kind] || defaultStyle;
```

Comment:
This doesn't resolve the dynamic part, does it ?
You still load once a style object